ETC.tsv

```diff
@@ -184,11 +184,11 @@ ETC_20150317_000183	Sauga
 ETC_20150317_000184	Saugas
 ETC_20150317_000185	Scorpion
 ETC_20150317_000186	Tontulia
-xETC_20150317_000187	Tontulia is a very vigorous mutated monster.
+xETC_20150317_000187	Tontulia is a very vigorous, mutant monster.
 ETC_20150317_000188	Tontus
-xETC_20150317_000189	The lives of residents in a dry and harsh environment was further worsened when the cacti mutated to Tontus.
+xETC_20150317_000189	The lives of residents in the dry and harsh environment were further impacted when the cacti mutated to Tontus.
```

Comment:
I would stick with "worsened" or some word that denotes a decrease in the quality of their lives, not simply a change. "Further Impacted" implies that there's some other impact the players know about that affected the residents, and we don't know that.